.gitlab-ci.yml

```diff
@@ -16,7 +17,7 @@ stages:
       - unknown_failure
 
 .image: &image
-  image: node:8-jessie
+  image: node:12.10
```

Comment:
You can use `erbium` here, that's what we also use in the gitlab runner node image - then it'll just be upgraded. Also, this will pull immediately from dockerhub, rather than through the dockerhub mirror: https://treehouse.surveymonkey.com/pages/viewpage.action?pageId=410093331

So you would end up with `registry.vox.dev/dockerhub/library/node:erbium`

Reply:
you right! Done